frontend/test/__support__/e2e/cypress-plugins.js

```diff
@@ -54,6 +56,9 @@ module.exports = (on, config) => {
    **                          CONFIG                                **
    ********************************************************************/
 
+  if (!isQaDatabase) {
+    config.ignoreTestFiles = "**/metabase-db/**";
+  }
   config.env.HAS_ENTERPRISE_TOKEN = hasEnterpriseToken;
 
```

Comment:
If you're interested in observing the resolved config, you can `console.log(config)` here.

Reply:
For the `yarn test-cypress-open`, you can access the resolved config by clicking on the "Settings" tab and then opening "Configuration"
![image](https://user-images.githubusercontent.com/31325167/126196486-51de3736-5950-4565-8a10-b018b242778b.png)